include/inviwo/core/properties/propertyowner.h

```diff
@@ -153,39 +154,31 @@ class IVW_CORE_API PropertyOwner : public PropertyOwnerObservable,
     // allocated on the heap.
     std::vector<std::unique_ptr<Property>> ownedProperties_;
 
+    void forEachProperty(std::function<void(Property&)> callback,
+                         bool recursiveSearch = false) const;
```

Comment:
Const?

Reply:
all the other property getter are const...